src/workloads/contrib/qe_range_testing/README.md

```diff
@@ -0,0 +1,34 @@
+# Summary
+
+qe_range_testing is a python program which generates Genny workloads for Queryable Encryption Range.
+
+## Prerequisites
+
+Install Poetry, following by [their documentation](https://python-poetry.org/docs/#installation) for one of the supported installation methods.
+
+Then, using this directory as your current working directory, invoke Poetry to install the required dependencies into your local virtual environment, using the specific versions enumerated in `poetry.lock`.
+
+```sh
+poetry install
+```
+
+## General Usage
+
+Note: For local development, you will have to edit the `MONGO_CRYPT_PATH` constant in `experiment_generator.py` to point to `mongo_crypt_v1.so` on your local machine. The easiest way to get this file is to run the `crypt_create_lib` task in the `Amazon Linux 2 arm64 Crypt Compile` build variant and download the resulting shared library file. Otherwise, you'll have to compile the library yourself; instructions not attached.
```

Comment:
I hesitate to spend the time on automating this since this step is only necessary if someone wants to regenerate the workload files and run them locally -- the shared library is not necessary for testing with Evergreen. I can make that more clear in the README.

Reply:
I was thinking that it could be helpful at least for reproducibility purposes. I don't anticipate that we will sorely miss the ability to generate workloads that can be run locally, but given the chance that we might want that functionality in the future, and I'm checking the Evergreen workload generation code in anyways, it doesn't seem prudent to remove the local generation code.